packages/shared/clickhouse/scripts/seed.ts

```diff
@@ -65,7 +65,6 @@ async function main() {
   } catch (error) {
     console.error("Error during Clickhouse preparation:", error);
   } finally {
-    await clickhouseClient().close();
```

Comment:
Redundant as this only closes the client we just created since there isn't a singleton.